### ISSUE TITLE
Is this code an adaption from ip_basic?

### ISSUE THREAD

**Issue by andreaceruti** [GCaptainNemo/depth-image-completion#2]
Hi! Thanks for sharing your work.
I have seen that you have listed as a reference [this repo](https://github.com/kujason/ip_basic). I have seen that in ip_basic we have a file called [depth_completion.py](https://github.com/kujason/ip_basic/blob/master/demos/depth_completion.py). Since running both of them I get a similar result, is this repo an adaption of that code? Or could you tell me what is the main difference?

**Comment by GCaptainNemo** [comment]
> Hi! Thanks for sharing your work. I have seen that you have listed as a reference [this repo](https://github.com/kujason/ip_basic). I have seen that in ip_basic we have a file called [depth_completion.py](https://github.com/kujason/ip_basic/blob/master/demos/depth_completion.py). Since running both of them I get a similar result, is this repo an adaption of that code? Or could you tell me what is the main difference?

Yes, this repo is an adaption of that code. I completely referred to its papers and [code](https://github.com/kujason/ip_basic/blob/master/demos/depth_completion.py), and implemented it in C++.
